tools/wptrunner/wptrunner/wptlogging.py

```diff
@@ -28,6 +29,22 @@ def setup_stdlib_logger():
     logging.root = stdadapter.std_logging_adapter(logging.root)
 
 
+def set_default_log_formatter(kwargs):
+    """When --verify is set and no default log formatter was set,
+    set a default log formatter to mach and its value is sys.stdout.
+    This is useful for ./wpt run --verify."""
+    log_formatters = ("raw", "unittest", "xunit", "html", "mach", "tbpl",
+                      "grouped", "errorsummary")
+    is_formatter_set = False
+    if kwargs.get("verify"):
+        for log_formatter in log_formatters:
+            if kwargs.get("log_" + log_formatter):
+                is_formatter_set = True
+                return
```

Comment:
The early `return` makes the flag unnecessary

Reply:
I'll remove the unnecessary line.